agenthub/browsing_agent/browsing_agent.py

```diff
@@ -97,14 +98,15 @@ class BrowsingAgent(Agent):
     def __init__(
         self,
         llm: LLM,
+        config: AgentConfig,
```

Comment:
Looks like this PR would have a lot of conflicts with mine: https://github.com/OpenDevin/OpenDevin/pull/2756 :)

Reply:
Hey @li-boxuan , let's merge yours first then. This one is still not close to being complete.